dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowCache.java

```diff
@@ -30,13 +30,16 @@
 	abstract protected WorkflowTask addTask(Contentlet contentlet, WorkflowTask task);
 	abstract protected WorkflowStep add(WorkflowStep step);
 	abstract protected List<WorkflowAction> addActions(WorkflowStep step, List<WorkflowAction> actions);
+	abstract protected List<WorkflowAction> addActions(WorkflowScheme scheme, List<WorkflowAction> actions);
 	abstract protected List<WorkflowAction> getActions(WorkflowStep step);
+	abstract protected List<WorkflowAction> getActions(WorkflowScheme scheme);
 	
 	abstract public void clearCache();
 	abstract protected void remove(Contentlet contentlet);
 	public abstract void remove(WorkflowScheme scheme);
 	abstract public void remove(WorkflowStep step);
 	abstract protected void removeActions(WorkflowStep step);
+	abstract protected void removeActions(WorkflowScheme scheme);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)